recipes/CommonVoice/ASR/transducer/README.md

```diff
@@ -2,14 +2,29 @@
 This folder contains scripts necessary to run an ASR experiment with the CommonVoice dataset: [CommonVoice Homepage](https://commonvoice.mozilla.org/)
 
 # Extra-Dependencies
-This recipe support two implementation of Transducer loss, see `use_torchaudio` arg in Yaml file:
-1- Transducer loss from torchaudio (if torchaudio version >= 0.10.0) (Default)
-2- Speechbrain Implementation using Numba lib. (this allow you to have a direct access in python to the Transducer loss implementation)
-Note: Before running this recipe, make sure numba is installed. Otherwise, run:
+This recipe supports three implementations of the transducer loss, see
+`framework` arg in the yaml file:
+1. Transducer loss from torchaudio (this requires torchaudio version >= 0.10.0)
+(Default).
+2. Speechbrain implementation using Numba. To use it, please set
+`framework=speechbrain` in the yaml file. This version is implemented within
+SpeechBrain and  allows you to directly access the python code of the
+transducer loss (and directly modify it if needed).
+3. FastRNNT (pruned / unpruned) loss function.
+  - To use the un-pruned loss function, please set `framework=fastrnnt`.
+  - To use the pruned loss function, please change the whole `transducer_cost`
+  yaml variable.
+
+If you are planning to use speechbrain RNNT loss function, install `numba`:
 ```
 pip install numba
 ```
 
+If you are planning to use FastRNNT loss function, install `FastRNNT`:
+```
+pip install FastRNNT
```

Comment:
Shouldn't this be `fast-rnnt`? I get a 404 on PyPI when checking for `FastRNNT`.